test/libdnf5/transaction/test_transaction.cpp

```diff
@@ -52,7 +52,8 @@ create_getter(new_transaction, &libdnf5::transaction::TransactionHistory::new_tr
 }  //namespace
 
 static Transaction create_transaction(libdnf5::Base & base, int nr) {
-    auto trans = (*(base.get_transaction_history()).*get(new_transaction{}))();
+    auto history = std::make_unique<libdnf5::transaction::TransactionHistory>(base);
```

Comment:
Is the `unique_ptr` needed here?

Reply:
As these are tests, I just applied the first idea that came to my mind. Is there a better alternative?

Reply:
Oh I see what you mean, just create an object on stack right?